aeproject-cli/aeproject-init/artifacts/docker/aeternity.yaml

```diff
@@ -21,15 +18,6 @@ keys:
   peer_password: "top secret"
   dir: ./keys
 
-chain:
-  persist: true
-  hard_forks:
-    "1": 0
-    "2": 1
-    "3": 2
-    "4": 3
```

Comment:
shouldn't this still be here to use the latest protocol?

also persistence for when using the same container again

Reply:
will check that, don't think so. persistence will be added